plugins/osdn/subnets.go

```diff
@@ -49,7 +51,12 @@ func (oc *OsdnController) addNode(nodeName string, nodeIP string) error {
 		return fmt.Errorf("Error writing subnet %v to etcd for node %s: %v", sn, nodeName, err)
 	}
 
-	log.Infof("Created HostSubnet %v for node %s (%s)", sn, nodeName, nodeIP)
+	sub, err := oc.Registry.GetSubnet(nodeName)
+	if err != nil {
+		return fmt.Errorf("Error fetching subnet for node %q: %v", nodeName, err)
+	}
```

Comment:
Unnecessary etcd call just for logging. Instead we could make CreateSubnet() to return the subnet along with the error (currently we are ignoring the returned subnet: https://github.com/dcbw/openshift-sdn/blob/more-hostsubnet-logging/plugins/osdn/registry.go#L105)